migrate.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS `vehicle_financing` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `citizenid` varchar(50) DEFAULT NULL,

```

Comment:
Do we need the citizenid? Could go through the vehicles table to inner join all vehicles associated to a citizenid with a finance entry.

Reply:
No need to get the both financing & vehicle data at the same time,
So it's easier to use citizenId as an id